ur_gazebo/package.xml

```diff
@@ -29,6 +29,7 @@
   <exec_depend>position_controllers</exec_depend>
   <exec_depend>robot_state_publisher</exec_depend>
   <exec_depend>ur_description</exec_depend>
+  <exec_depend>gazebo_plugins</exec_depend>
```

Comment:
I think we should also add: `gazebo-ros-pkgs` 
It is not coming installed by default.

Reply:
I tried adding it and in my case rosdep was not able to locate the package (even though it is there). This is weird. Can you try also on your end (e.g., by removing the package and trying to install through rosdep install)? 
If also does not work, we need to include it under `readme` so the users manually installs.

Reply:
I think we should at least add `<exec_depend>gazebo-ros-pkgs</exec_depend>` as that package is required in the execution to publish ft_sensor as a rostopic

Reply:
Actually rosdep uses some kind of dictionary or mapping, and those use snake naming. I will try `gazebo_ros_pkgs`, should work.

Reply:
We should test against a fresh ROS Noetic install

Reply:
`gazebo_ros_pkgs` in `package.xml` works!